src/DatabaseSchemaFacade.php

```diff
@@ -0,0 +1,18 @@
+<?php
```

Comment:
Do we actually need a Facade for this? The Facade is only useful if you want to `use DatabaseSchema;` instead of `use Backpack\CRUD\DatabaseSchema`, correct?

If we're the only ones using it... no point in adding a Facade for it, right? And not adding a Facade should keep the impact smaller on what we register, right?

Reply:
The reason we use a facade is so that we can call non static methods statically!